C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -170,15 +171,12 @@ public async Task<object> ValidateOauthCodeAsync(string code)
         /// <returns>A Slack message object with {text, attachments, channel, thread ts} as well as any fields found in activity.channelData</returns>
         public object ActivityToSlack(Activity activity)
         {
-            var channelId = activity.Conversation.Id;
-            var threadTS = (activity.Conversation as dynamic).threadTS;
-
             dynamic message = new ExpandoObject();
             message.TS = activity.Id;
             message.Text = activity.Text;
             message.Attachments = activity.Attachments;
-            message.Channel = channelId;
-            message.ThreadTS = threadTS;
+            message.Channel = activity.Conversation.Id;
+            message.ThreadTS = (activity.Conversation as dynamic).threadTS;
```

Comment:
`Activity` does not seem to have a property named `threadTS` and since its not an `object` type it won't have said property.
If in NodeJS said property is assigned to the `activity` class, then we need to revise this and extend/create a new class that include this property

Reply:
If this is requires to extend more classes, perhaps we can leave this for when we attack the _dynamics_ issue?

Reply:
Yes, lets leave this as it is for now. Once we merge everything lets fix it on a single pull request.